docs/user/managing-flux-packages.md

```diff
@@ -0,0 +1,210 @@
+
+### Creating a service account
+
+Since the Flux system reconciles a `HelmRelease` in the background, we need to create a service account to use when creating a Helm release via Flux. This service account is created in the namespace where you intend to install the package (specifically, where the HelmRelease object will be created).
+
+```bash
+cat > kubeapps-user-service-account.yaml << EOF
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: flux-reconciler
+  namespace: kubeapps-user-namespace
+automountServiceAccountToken: false
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: flux-reconciler
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: flux-reconciler
+  namespace: kubeapps-user-namespace
+roleRef:
+  kind: ClusterRole
+  name: edit
+  apiGroup: rbac.authorization.k8s.io
+EOF
+
+kubectl apply -f kubeapps-user-service-account.yaml
+```
+
+### Installing a Package
+
+Installing a Flux Package in Kubeapps is very similar to installing any other package (such as a Helm Chart) in Kubeapps.
+
+> **TIP**: Please refer to the [user documentation](./dashboard.md) for more information on how to use Kubeapps as a user.
+
+Assuming that a Flux `HelmRepository`, such as the Bitnami repository above, is already installed in the cluster, you can simply go to the `Catalog` tab and select the package you want to install.
+
+The following image depicts the catalog page with a set of packages from the above Bitnami `HelmRepository`.
+
+![Catalog page with Flux Packages](../img/flux-catalog.png)
+
+Next, select any package you want to install, for example, `Apache`, then click Deploy to see the install form as depicted below:
+
+![Installation page of a Carvel Package](../img/flux-install.png)
+
+A big difference with respect to other packaging formats is that **you must select a `ServiceAccount` to be used for installing the package**. This is because Flux, similar to [Carvel](managing-carvel-packages.md), will carry out the installation as well as upgrades in the background and so cannot rely on doing so as the user. See [Creating a service account](#creating-a-service-account) above.
+
+In Kubeapps, a dropdown will allow you to select which `ServiceAccount` you want to use, as shown in the above screenshot.
+
+> **NOTE**: As a consequence, the user logged in Kubeapps will need RBAC permissions to perform a `list` operation on `ServiceAccount` objects.
+
+> **TIP**: For the Apache package, ensure that the ServiceType is set to an option supported by your cluster. For instance, for a local Kind cluster, you will want to choose `ClusterIP` rather than `LoadBalancer`.
+
+> **TIP**: you can uncomment a set of lines easily by selecting them and typing `Ctrl + /`.
```

Comment:
Not sure if this also applies to the Flux plugin case, as I assume these charts also have a set of default values, no (rather than being artificially generated, like in Carvel)